build.sh

```diff
@@ -38,7 +53,12 @@ menu_option_1() {
 
   ### Start the Build. ###
   echo "Starting the build...."
-  packer build -force \
+  if [ "$debug" = true ]
+  then
+    packer build -force -on-error=ask -debug "$INPUT_PATH"
```

Comment:
Looks like we're double including $INPUT_PATH in all of these build commands now? Once at the start here and then once after (line 67 in this one)

Reply:
👀 Great catch! Looks like that's always been there and needs to be adjusted!

Reply:
Resolved in https://github.com/vmware-samples/packer-examples-for-vsphere/commit/992d77ee0693f733be68cbde6d0b6bc454a630a6.